.env.dev

```diff
@@ -1,4 +1,5 @@
 RUNSERVER_PORT=8080
+COMPOSE_PROFILES=with_grafana
```

Comment:
this was referenced in `README.md`. It mentioned to manually add this to a `.env`. I added a note about how to disable this in `DEVELOPER.md`.